src/main/java/org/apache/commons/lang3/exception/ExceptionUtils.java

```diff
@@ -936,4 +944,4 @@ public static void removeCommonFrames(final List<String> causeFrames, final List
     public ExceptionUtils() {
         super();
     }
-}
+}
```

Comment:
Please don't make unnecessary whitespace changes.

Reply:
I don't understand this diff, the old and the new versions have the exact same '}' without any whitespace change.

Reply:
If you hover over the red icon you see `No newline at end of file`. So the original had a newline at the end and it was removed in your change.